app/models/csv_updater.rb

```diff
@@ -0,0 +1,389 @@
+class CsvUpdater
```

Comment:
Does this need to be in `models/` or can we put it in helpers?

Reply:
It can be put into helpers. I'll make this change